Formula/git-imerge.rb

```diff
@@ -15,13 +17,71 @@ class GitImerge < Formula
     sha256 "76aee24eeb5e7615e4cfbd7aaf3aacac8d8729dfcee79d8542a84cbd9b663459" => :el_capitan
   end
 
+  depends_on "python@3.8"
+
+  # remove the custom completiondir process
+  patch :DATA
+
   def install
-    bin.mkpath
-    # Work around Makefile insisting to write to $(DESTDIR)/etc/bash_completion.d
-    system "make", "install", "DESTDIR=#{prefix}", "PREFIX="
+    virtualenv_install_with_resources
+
+    bash_completion.install "completions/git-imerge"
   end
 
   test do
-    system bin/"git-imerge", "-h"
+    system "git", "init"
+    system "git", "config", "user.name", "BrewTestBot"
+    system "git", "config", "user.email", "BrewTestBot@example.com"
+
+    system "echo \"puts 'Hello, world'\" > main.rb && git add main.rb && git commit -m \"First commit\""
```

Comment:
Could this be split into separate system calls? And also use `(testpath/"file").write`?

Reply:
👍